Sources/LoggingFormatAndPipe/BasicFormatter.swift

```diff
@@ -72,6 +78,12 @@ public struct BasicFormatter: Formatter {
     )
 
     /// Adorkable's go-to log format 😘
+    ///
+    /// `{timestamp} ▶ {level} ▶ {file}:{line} ▶ {function} ▶ {message} ▶ {metadata}`
+    ///
+    /// *Example:*
+    ///
+    /// `2019-07-30T13:49:07-0400 ▶ error ▶ /asdf/swift-log-format-and-pipe/Tests/LoggingFormatAndPipeTests/FormatterTests.swift:25 ▶ testFormatter(_:) ▶ Test error message`
```

Comment:
Great, thanks a lot for including examples :) Much easier to pick a formatter now